workflows/rnaseq.nf

```diff
@@ -26,8 +36,7 @@ checkPathParamList = [
 ]
 for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true) } }
 
-// Check mandatory parameters
-if (params.input) { ch_input = file(params.input) } else { exit 1, 'Input samplesheet not specified!' }
```

Comment:
shouldn't you keep that for now?

Reply:
I believe it should be happening from here now:
https://github.com/nf-core/rnaseq/blob/9a47abe0bf7762f62eaa945c7786f181a2befe63/nextflow_schema.json#L18

But, you are right. I will leave it in for now until I have tested that properly.